src/explore-education-statistics-common/src/modules/charts/components/MapBlockInternal.tsx

```diff
@@ -417,6 +417,14 @@ export const MapBlockInternal = ({
   const selectedDataSetConfiguration =
     dataSetCategoryConfigs[selectedDataSetKey];
 
+  const selectedDataSet =
+    selectedFeature?.properties?.dataSets[selectedDataSetKey];
+
+  const {
+    config: selectedDataSetConfig,
+    dataSet: expandedSelectedDataSet,
+  } = dataSetCategoryConfigs[selectedDataSetKey];
```

Comment:
We already assign `selectedDataSetConfiguration` from `dataSetCategoryConfigs[selectedDataSetKey]`. This would  be easily confused with the new destructured `selectedDataSetConfig` variable.

I'd probably lean towards getting rid of this destructuring and just using `selectedDataSetConfiguration` (maybe we want to rename it to `selectedDataSetConfig` for brevity).

Reply:
yep, good spot. sorted.